static/app/gettingStartedDocs/javascript/angular.spec.tsx

```diff
@@ -72,4 +72,17 @@ describe('javascript-angular onboarding docs', function () {
       screen.getByText(textWithMarkupMatcher(/replaysOnErrorSampleRate: 1\.0/))
     ).toBeInTheDocument();
   });
+
+  it('enables profiling by setting profiling samplerates', () => {
+    renderWithOnboardingLayout(docs, {
+      selectedProducts: [ProductSolution.ERROR_MONITORING, ProductSolution.PROFILING],
```

Comment:
Not sure this is a possible configuration because we require `ProductSolution.PERFORMANCE` along side `ProductSolution.PROFILING` for the time being

Reply:
True, but iirc the UI just has a single checkbox to enable profiling. We'd need to make it disabled until folks enable perf